src/PhpSpec/Console/Application.php

```diff
@@ -57,9 +58,17 @@ public function getContainer()
      */
     public function doRun(InputInterface $input, OutputInterface $output)
     {
+        $helperSet = $this->getHelperSet();
         $this->container->set('console.input', $input);
         $this->container->set('console.output', $output);
-        $this->container->set('console.helper.dialog', $this->getHelperSet()->get('dialog'));
+        $this->container->set('console.helper.dialog', $helperSet->get('dialog'));
```

Comment:
Currently the Application will always get the dialogHelper. I had not realised that it was deprecated that strongly. Would it be too much of a BC break to remove it from the constructor of the Application? Otherwise I could look at injecting a Proxy of some sort.